src/components/Banners/SiteBanner.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from "react";
+import { useQuery } from "@apollo/react-hooks";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import IconButton from "@leafygreen-ui/icon-button";
+import Icon from "@leafygreen-ui/icon";
+import Cookies from "js-cookie";
+import get from "lodash.get";
+import { GET_SITE_BANNER } from "gql/queries";
+import { SiteBannerQuery } from "gql/generated/types";
+
+const { yellow, blue, green, red } = uiColors;
+export const SiteBanner = () => {
+  const { data, loading } = useQuery<SiteBannerQuery>(GET_SITE_BANNER);
+  const { text, theme } = get(data, "siteBanner", {});
```

Comment:
tldr: consider using null propagation and nullish coalescing instead of `get`, which would be
```
const siteBanner = data?.siteBanner
const text = siteBanner?.text ?? ""
const theme = siteBanner?.theme ?? ""
```

I think we should move away from using `get` because it prevents us from being able to utilize TS autocomplete when accessing properties on the data object. using null propagation and nullish coalescing will yield same result as `get` but with type safety. for example, if `data.siteBanner` were to change to `data.banner`, this line of code would not signify an error and would just return the empty object.

Reply:
Great idea!